### DISCUSSION TITLE
Can't connect GitHub organization with Vercel team

### DISCUSSION THREAD

**Discussion by nhuesmann** [vercel/vercel#5349] [Help] [answered]
Hello! I've recently moved my company related repos to a GitHub organization. I understand Vercel requires a team in order to connect to GH orgs, so I've created a team and am on the 14 day trial. So the setup is that I have a personal hobby account which is the owner of a team.

When I try to connect to GitHub via Vercel, I enter the repo url (which is a private repo within my GH organization), and am met with the following error:

`Error: The GitHub account "cardboardcompanion" doesn't exist, please check again.`

How should I proceed?

**Comment by grinono** [comment]
i'm having the same issue.

**Comment by tomohiro-n** [comment]
Same here...

**Reply by tomohiro-n** [reply]
tried it again and found it's fixed. Thanks!

**Comment by paulogdm** [comment, marked as answer]
Hello,

To investigate the issue further, please send us a ticket on https://vercel.com/support/request.

Let us know.

**Comment by tupizz** [comment]
same here
